releasenotes/notes/farewell-retworkx-be18546ac0b0cbc2.yaml

```diff
@@ -0,0 +1,15 @@
+---
+deprecations:
+  - |
+    We are saying goodbye to the legacy ``retworkx`` package that operates as a
+    backwards compatibility alias for ``rustworkx``. It was originally marked as
+    deprecated and this will be its last release. The ``rustworkx`` name has now existed
+    longer than the ``retworkx`` name and the ``retworkx`` package is no longer needed.
```

Comment:
It's almost that long right but not quite? It looks like we released rustworkx in 0.12.0 which was tagged Oct 7, 2022. The first retworkx release was 0.0.1 in May, 2019. So we're still a few  months short of that crossover point.

Although I'd agree that this is probably arguably true because those super early releases were unusable and you couldn't really start to use the package until about 0.3.0 which was released on Feb 20, 2020. So I'm fine keeping this. More was just excited to start discussing history :)

Reply:
There's also a delay on when we release 0.17, so I think by them it will be pretty much even! We can always add a `nearly as long` if you think that is necessary